doc/sphinx/scripting/python/fontforge.rst

```diff
@@ -4910,6 +4910,13 @@ See the :class:`selection` type for how to alter the selection.
    Leaves only areas in the intersection of contours in all selected glyphs.
    See also :meth:`font.removeOverlap()`.
 
+.. method:: font.isWorthOutputting()
+
+   Returns whether the current selection is worth outputting into a font file.
+   Basically a glyph is worth outputting if it contains any contours, or
+   references or has had its width set.
+   See also :meth:`glyph.isWorthOutputting()`.
```

Comment:
This is good, but what happens if the font is just opened and nothing is selected? It seems wrong that `font.isWorthOutputting()` would consider 0 glyphs instead of all the ones in the font by default, and that's not done for any other function.

`font.isWorthOutputting` should only consider the selection if one is set. No one will ever want to know if 0 glyphs are worth outputting.

Reply:
It has been a while but I think this is the case - `select.none()` causes `isWorthOutputting()` to return false (it is part of the test case).

Reply:
Right, that's the issue.

Reply:
The behavior you requested is already implemented, and as tested/verified around line 34 of the test case on `select.none()`.